src/sb_pointcloud_processing/src/test_pcl_generator_node.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Created by: Min Gyo Kim
+ * Created On: March 11th 2018
+ * Description: Generates a point cloud with two lines and an optional outlier
+ * line
+ *              and publishes it to "/input/pointcloud"
+ */
+
+#include "../test/TestUtils.h"
```

Comment:
We should change this so that it automatically searches from the top level rather then from this folder (mostly as a matter of convention). ie. `#import "test/TestUtils.h"`

Reply:
done